docs/mergeduty/howto.md

```diff
@@ -274,18 +275,17 @@ python mozharness/scripts/merge_day/gecko_migration.py \
 
 :warning: The decision task of the resulting pushlog in the `mozilla-beta` might fail in the first place with a timeout. A rerun might solve the problem which can be caused by an unlucky slow instance.
 
+### Re-opening the tree(s)
+
+kalk to Sheriffs and RelMan to re-open trees (ether `open` or `approval-only`) so that l10n bumper can run.
```

Comment:
tiny nits

s/kalk to/ask/ ?

s/ether/either

Reply:
Good catch, fixed both in a follow-up.